### DISCUSSION TITLE
exe app installer not in windowstore

### DISCUSSION THREAD

**Discussion by quoccc** [File-New-Project/EarTrumpet#980] [Ideas]
because my window store cant use. so can you make a installer exe ver? i will very appreciate that.(sorry for my bad english)

**Comment by EvilGremlin** [comment]
Second that, i delete store along with all other bloatware and stock apps on all my installs. To not have standalone installer (with silent install keys) is great omission for any app.

**Reply by stbcoding** [reply]
Can you please provide an .exe for install?
I would really appreciate it.
Thanks!

**Comment by riverar** [comment]
Hi @quoccc! We also have EarTrumpet on Chocolatey and winget. (And you can download the chocolatey package manually, if you'd like the raw files.) See https://eartrumpet.app for links.

We are not creating/release our own installer, sorry.

More details: https://github.com/File-New-Project/EarTrumpet/issues/1004#issuecomment-1010675687

**Comment by wiebenor** [comment]
Although I don't want to use the store version myself, nor have I heard of these other ways to download it, I think I understand your reasoning for doing it that way. One thing I would like to point out though, is that I would love to have something like this, in a pre-install environment, which you may or may not know, is famous for not supporting anything but EXE installers, and even EXE installer support is modded in usually, but I thought I'd postulate that as being a reason for an executable, albeit a niche usage case. Speaking of chocolatey though, what exactly is chocolatey, and what exactly do you mean by "raw files"??? I could possibly get this to work for my needs, if by "raw files" you mean like, a flatpack, or portable, style installation, where you just click on an EXE to run the thing. Of course, running it may only work in full windows, but still, even so, I would still have a use for it, if it IS similar to a flatpack or portable install...

**Reply by jac0-b** [reply]
Chocolatey is just a package manager for windows. Once you have chocolatey it's as simple as `choco install eartrumpet`
